### PR TITLE
feat: move kyc to events

### PR DESCRIPTION
**Pull request by Tguntenaar**
_Resolves #848_

This PR moves the kyc and new terms and conditions to the events committee portal.

## Relates to [144](https://github.com/NEAR-DevHub/neardevhub-contract/issues/144)

We need to update the front end together witht the indexer and contract changes.

## TODO

- [x] Update the contract and deploy
- [x] Deploy the newest indexer
- [ ] Add github action to deploy front end

### CODE REVIEW

**Comment by Tguntenaar** [comment]
@race-of-sloths include

**Comment by race-of-sloths** [comment]
@Tguntenaar Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
You are true leader! Other sloths hot on your heels, keep moving!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/Tguntenaar?type=bot&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/Tguntenaar?type=bot&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/Tguntenaar)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command
</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/Tguntenaar)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

File: instances/events-committee.near/widget/devhub/entity/proposal/AcceptedTerms.jsx
Comment (Megha-Dev-19) [inline instances/events-committee.near/widget/devhub/entity/proposal/AcceptedTerms.jsx:83]: does events have different terms and condition?
I thought it was same?
but either way, we can pass this `href` as prop and also the feed indexer query and then can use the same component, or m I missing something?

File: instances/events-committee.near/widget/devhub/entity/proposal/CommentsAndLogs.jsx
Comment (Megha-Dev-19) [inline instances/events-committee.near/widget/devhub/entity/proposal/CommentsAndLogs.jsx:334]: you added `AcceptedTerms` to events instance, but here we're using devhub?

File: instances/events-committee.near/widget/devhub/entity/proposal/Proposal.jsx
Comment (Megha-Dev-19) [inline instances/events-committee.near/widget/devhub/entity/proposal/Proposal.jsx:348]: before merging this, we should also deploy the latest changes to events contract, else it might throw error.

Comment (Tguntenaar) [reply instances/events-committee.near/widget/devhub/entity/proposal/Proposal.jsx:348]: Yes for sure. Update the indexer as well, see this issue: https://github.com/NEAR-DevHub/neardevhub-contract/issues/144

File: instances/events-committee.near/widget/devhub/entity/proposal/AcceptedTerms.jsx
Comment (Tguntenaar) [reply instances/events-committee.near/widget/devhub/entity/proposal/AcceptedTerms.jsx]: Since they have the same T&C I have removed this file and reused the accepted terms from DevHub. We don't need the pass href since that component will be the same.

**Review by Megha-Dev-19 — APPROVED** [review]

**Comment by Megha-Dev-19** [comment]
@race-of-sloths score 3

**Comment by race-of-sloths** [comment]
### 🌟 Score recorded!

@Megha-Dev-19, thank you for scoring this pull request in the Race of Sloths!

**Comment by Tguntenaar** [comment]
It seems like a test failed after the `main` branch got merged into this branch. Both branches didn't have any tests failing but I will revert the merge.

**Comment by race-of-sloths** [comment]
### ✅ PR is finalized! 
Your contribution is much appreciated with a final score of 3!
You have received 30 Sloth points for this contribution


We would appreciate your feedback or contribution idea that you can submit [here](https://github.com/NEAR-DevHub/race-of-sloths/issues/new/choose)

**Comment by petersalomonsen** [comment]
@race-of-sloths score 3

**Comment by race-of-sloths** [comment]
### ⏰️ Scoring closed
24 hours have already passed since the merge

**Pull request closed without merging**
